www/docs/en/dev/guide/appdev/whitelist/index.md

```diff
@@ -19,6 +19,7 @@ license: >
 
 title: Whitelist Guide
 toc_title: Whitelisting
+description: Grant an application access to external resources.
```

Comment:
Securely grant an application... Just trying to emphasize that this is a security feature.